<launch>
    <arg name="rgb_topic" default="/rgb_topic" />
    <arg name="depth_topic" default="/depth_topic" /> 
    <arg name="camera_topic" default="/camera_info" />
    <arg name="odom_topic" default="/odom_topic" />
    <node pkg="sfm" type="sfm" name="structure_from_motion" required="true" output="screen" clear_params="true">
      <param name="rgb_topic" type="string" value="$(arg rgb_topic)" />
      <param name="depth_topic" type="string" value="$(arg depth_topic)" />
      <param name="camera_topic" type="string" value="$(arg camera_topic)" />
      <param name="odom_topic" type="string" value="$(arg odom_topic)" />
    </node>
</launch>
